src/Appium.Net/Appium/AppiumDriver.cs

```diff
@@ -125,63 +125,63 @@ public AppiumDriver(AppiumLocalService service, ICapabilities appiumOptions, Tim
         #region Generic FindMethods
 
         public new W FindElement(By by) =>
-            (W) base.FindElement(by);
+            (W)base.FindElement(by);
 
         public new ReadOnlyCollection<W> FindElements(By by) =>
             ConvertToExtendedWebElementCollection<W>(base.FindElements(by));
 
-        public new W FindElement(string by, string value) => (W) base.FindElement(by, value);
+        public new W FindElement(string by, string value) => (W)base.FindElement(by, value);
 
         public new IReadOnlyCollection<W> FindElements(string selector, string value) =>
             ConvertToExtendedWebElementCollection<W>(base.FindElements(selector, value));
 
-        public new W FindElementByClassName(string className) =>
-            (W) base.FindElement(MobileSelector.ClassName, className);
+        public W FindElementByClassName(string className) =>
```

Comment:
these helpers actually got [removed](https://github.com/appium/java-client/pull/1531) from the java client. Perhaps you could sync with @valfirst on this topic?